Merge latest changes from upstream repo (#7)
LiskHQ/lisk-node: README.md

1. Ensure you have an Ethereum L1 full node RPC available (not Lisk), and set the `OP_NODE_L1_ETH_RPC` and the `OP_NODE_L1_BEACON` variables (within the `.env.*` files, if using docker-compose). If running your own L1 node, it needs to be synced before the Lisk node will be able to fully sync.
2. Please ensure that the environment file relevant to your network (`.env.sepolia`, or `.env.mainnet`) is set for the `env_file` properties within `docker-compose.yml`. By default, it is set to `.env.mainnet`.

> 3. Please ensure to set `CLIENT` before starting the node (available options are geth and reth). By default, it is set to `geth`. 

Reviewer: ```suggestion
3. We currently support running either the `op-geth` or the `op-reth` nodes alongside the `op-node`. By default, we run the `op-geth` node. If you would like to run the `op-reth` node instead, please set the `CLIENT` environment variable to `reth` before starting the node. 
```

Refer to the `op-geth` configuration [documentation](https://docs.optimism.io/builders/node-operators/management/configuration#op-geth) for detailed information about available options.

#### Run op-reth

Navigate to your `reth` directory and start service by running the command:

For, Lisk Sepolia Testnet:

```sh
./target/release/op-reth node \
  -vvv \
  --datadir="$DATADIR_PATH" \
  --log.stdout.format log-fmt \
  --ws \
  --ws.origins="*" \
  --ws.addr=0.0.0.0 \
  --ws.port=8546 \
  --ws.api=debug,eth,net,txpool \
  --http \
  --http.corsdomain="*" \
  --http.addr=0.0.0.0 \
  --http.port=8545 \
  --http.api=debug,eth,net,txpool \
  --authrpc.addr=0.0.0.0 \
  --authrpc.port=8551 \
  --authrpc.jwtsecret=PATH_TO_JWT_TEXT_FILE \

>   --metrics=0.0.0.0:606 \

Reviewer: ```suggestion
  --metrics=0.0.0.0:6060 \
```

  --chain=PATH_TO_NETWORK_GENESIS_FILE \
  --disable-discovery \

>   --rollup.sequencer-http=https://rpc.sepolia-api.lisk.com \

Reviewer: We could leave it as a variable but recommend the office sequencer endpoint from Lisk.

  --rollup.disable-tx-pool-gossip \
  --override.canyon=0
```

For, Lisk Mainnet:

```sh
./target/release/op-reth node \
  -vvv \
  --datadir="$DATADIR_PATH" \
  --log.stdout.format log-fmt \
  --ws \
  --ws.origins="*" \
  --ws.addr=0.0.0.0 \
  --ws.port=8546 \
  --ws.api=debug,eth,net,txpool \
  --http \
  --http.corsdomain="*" \
  --http.addr=0.0.0.0 \
  --http.port=8545 \
  --http.api=debug,eth,net,txpool \
  --authrpc.addr=0.0.0.0 \
  --authrpc.port=8551 \
  --authrpc.jwtsecret=PATH_TO_JWT_TEXT_FILE \

>   --metrics=0.0.0.0:606 \

Reviewer: ```suggestion
  --metrics=0.0.0.0:6060 \
```

  --chain=PATH_TO_NETWORK_GENESIS_FILE \
  --disable-discovery \

>   --rollup.sequencer-http=https://rpc.api.lisk.com \

Reviewer: Same as above.
